source/mainnet/index.rst

```diff
@@ -1,11 +1,66 @@
-:template: index.html
+==================================
+Concordium Developer Documentation
+==================================
```

Comment:
Yes, on source/mainnet/index all content should be removed except for the toctree at the bottom which controls the top menus. This is a common pattern in Sphinx documentation where the root index file primarily serves as a navigation controller rather than a content page. I forgot to delete the redundant content during restructuring.

Reply:
Fixed